Smarty/templates/DetailViewUI.tpl

```diff
@@ -95,9 +95,9 @@
                     							   <select id="txtbox_{$keyfldname}" name="{$keyfldname}" class="small" style="width:280px;">
                     								{foreach item=arr from=$keyoptions}
 											{if $arr[0] eq $APP.LBL_NOT_ACCESSIBLE}
-                    									<option value="{$arr[0]}" {$arr[2]}>{$arr[0]}</option>
+                    									<option label="{$arr[0]}" value="{$arr[0]}" {$arr[2]}>{$arr[0]}</option>
```

Comment:
Please explain to me the reason for adding the label attribute here. I don't think it is needed for anything